layer.go

```diff
@@ -99,6 +119,21 @@ type Layer struct {
 	Exec Exec `toml:"-"`
 }
 
+func (l Layer) BOMPath(bl BOMLocation, bt BOMFormat) string {
+	var bom string
+
+	switch bl {
+	case BuildBOM:
+		bom = "build"
+	case LaunchBOM:
+		bom = "launch"
```

Comment:
It seems a bit weird to me that the buildpack level boms would be attached to the layer. IMHO this should only return the layer specific bom path and the build contexts or the `Layers` object should contain the top level bom files.

Reply:
I can take a look at changing this. It ended up being pretty convenient when I was testing this out. Due to the layer contributor approach, you usually have a layer when you're going to write the SBoM. I can think of at least one case where this might not be true though, so let me play around a little more.

Excellent feedback, keep it coming :)

Reply:
OK, I can work with that in libpak. I submitted an update with this change. It moves build and launch SBoM methods to the Layers object.